src/serialization.hpp

```diff
@@ -205,7 +205,7 @@ Status serializePredictResponse(
         status = response->addOutput(
             outputInfo->getMappedName(),
             getPrecisionAsOVMSDataType(actualPrecision),
-            tensor.get_shape().data(),
+            reinterpret_cast<const int64_t*>(tensor.get_shape().data()),  // error handling ?
```

Comment:
Shouldn't we at this point validate tensor.get_shape() if it would not overflow?

Reply:
This is not an issue since we check in line 170 that tensor shape matches output shape metadata